addons/web/static/tests/helpers/test_utils_mock.js

```diff
@@ -91,7 +91,7 @@ async function _getMockedOwlEnv(params, mockServer) {
             },
         }, params.env.dataManager),
         services: Object.assign(services, params.env.services),
-        session: params.session || {},
+        session: params.env.session || params.session || {},
```

Comment:
yes.... this is not clear on our part either
we are not sure of the API that createStuff() should take
- either the existing: passing almost everything as first order keys of `params` (so: `params.session`)
- or passing stuff inside their own structure (so `params.env.session`)

I remember a talk with @aab-odoo that we wanted to take the direction of the second in the future (that way the developer knows what ends up where...)
But converting everything is out of scope of this PR
Nonetheless, the two ways would have to cohabitate

Reply:
I made this change because our `mail/test_env.js` has functions to generate env, and these functions only have `providedEnv` as param.